app/resources/admin.html.template

```diff
@@ -67,61 +67,83 @@
   function validate_settings() {
     //find all languages that the user specified with "Use?" checkboxes and
     //their associated titles.  Only allows titles from used languages.
+    var custom_message_table_ids = ['repo_titles', 'start_page_custom_htmls', 'results_page_custom_htmls', 'view_page_custom_htmls', 'seek_query_form_custom_htmls', 'footer_custom_htmls'];
```

Comment:
could you try to keep the line under 80 characters by inserting proper indentations? this is not a strict rule in html files, but it's still helpful for those developing in limited terminal window :)